### PR TITLE
Use custom hands model with the Hands component

### PR DESCRIPTION
**Pull request by AlaricBaraou**
This PR should allow user to load their own models when using the` <Hands />` component.

They can specify a model for the left or right hands individually like so : 

```
<Hands
      modelLeft={"https://vazxmixjsiawhamofees.supabase.co/storage/v1/object/public/models/left-hand-black-webxr-tracking-ready/model.gltf"}
      modelRight={"https://vazxmixjsiawhamofees.supabase.co/storage/v1/object/public/models/right-hand-black-webxr-tracking-ready/model.gltf"}
/>
```
If no model are specified it will load the current model.

Let me know if you want me to code it differently / rename stuff etc. 

Once it's merged, I'll try to make a simple demo of it and add it to the readme and getting started.

( Also, can we rename src\webxr\HandModel.js to something else ? Maybe I'm missing why it's called Oculus, because I don't see why it's specific to Oculus )

Thank you in advance !

### CODE REVIEW

File: src/webxr/XRHandMeshModel.js [hunk 1]
```diff
@@ -3,15 +3,17 @@ import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader'
...
     this.handModel = handModel
 
     this.bones = []
 
     const loader = new GLTFLoader()
+    
+    if(!customModel)
     loader.setPath(path || DEFAULT_HAND_PROFILE_PATH)
-    loader.load(`${handedness}.glb`, (gltf) => {
+    loader.load(customModel ? customModel : `${handedness}.glb`, (gltf) => {
```
Comment (sniok) [inline src/webxr/XRHandMeshModel.js:16]: this can be replaced with a 
```
customModel ?? `${handedness}.glb`
```

File: src/Hands.tsx [hunk 2]
```diff
@@ -2,18 +2,21 @@ import { useThree } from '@react-three/fiber'
...
+    modelLeft?:string
+    modelRight?:string
+  }) {
   const { scene, gl } = useThree()
 
   useEffect(() => {
     // @ts-ignore
     const hand1 = gl.xr.getHand(0)
-    hand1.add(new OculusHandModel(hand1))
+    hand1.add(new OculusHandModel(hand1,[props.modelLeft,props.modelRight]))
```
Comment (sniok) [inline src/Hands.tsx:14]: could you resolve the merge conflict here? it's just loop instead of each hand

**Comment by sniok** [comment]
Nice! Also would be nice to add some documentation to readme or I can do it later

**Comment by sniok** [comment]
and yeah it can be renamed to just `HandModel`, there's nothing 'oculus' about it

**Comment by sniok** [comment]
once we merge this and #63 we can do a "Hand" release :D

**Comment by AlaricBaraou** [comment]
I was thinking of a demo where you can change your hands dynamically but the way it's currently done it wouldn't work.

Would it be worth to make this dynamic ? 

I can see some use cases but there is not a lot of them.

**Comment by sniok** [comment]
we can keep it simple for now and add dynamic loading later

**Pull request merged**